frappe/model/rename_doc.py

```diff
@@ -146,6 +146,14 @@ def update_autoname_field(doctype, new, meta):
 		if field and field[0] == "field":
 			frappe.db.sql("UPDATE `tab{0}` SET `{1}`={2} WHERE `name`={2}".format(doctype, field[1], '%s'), (new, new))
 
+def update_child_autoname_field(doctype, link_field, new, old, meta):
```

Comment:
Inner functions should be inside the calling function

https://github.com/frappe/erpnext/wiki/Coding-Standards#function-sequence